birdhouse/components/weaver/post-docker-compose-up

```diff
@@ -335,21 +335,19 @@ echo "${PREFIX}All Weaver remote WPS providers registered successfully!"
 
 
 echo "${PREFIX}Starting Weaver WebApp/Worker Celery tasks validation..."
-CUR_SCRIPT_DIR="$(dirname "$(realpath "$0")")"
-PAVICS_COMPOSE="$(realpath "${CUR_SCRIPT_DIR}/../../pavics-compose.sh")"
 PAVICS_LOG_DIR="${PAVICS_LOG_DIR:-/tmp/pavics-compose}"
 CELERY_HEALTHCHECK="/opt/local/bin/weaver/celery-healthcheck"
 mkdir -p "${PAVICS_LOG_DIR}"
 # note: use 'tee' instead of capturing in variable to allow displaying results directly when running command
-${PAVICS_COMPOSE} exec weaver bash "${CELERY_HEALTHCHECK}" | tee "${PAVICS_LOG_DIR}/weaver.log"
+docker exec weaver bash "${CELERY_HEALTHCHECK}" | tee "${PAVICS_LOG_DIR}/weaver.log"
```

Comment:
We can still use `pavics-compose.sh`, especially now that the build directory is only re-created on "up". But I still think this is a good change here since it's no longer necessary to call `pavics-compose.sh` and this is simpler (and more consistent with what we're doing in other `post-docker-up` scripts.

Reply:
I didn't even realize this change in my previous review, but I think it can actually produce quite different behaviour. 
Using `docker exec` directly (instead of `docker compose exec`) will not apply all other configuration in docker compose such as mounted volumes, configs, networks, etc. Since the database network is not `default` here, it is possible this change actually breaks the healthcheck.

Reply:
`docker exec` runs a command in a container that is already created and already running. At the point where `docker exec` is called here, all volumes, networks, configs, etc. are already associated with the container. Even `docker compose exec` wouldn't apply any of the configurations you describe.

You can think of this as more or less the same as using `ssh` to execute a command on a machine that is already running.

Reply:
This assumes that `container_name` was not overridden, while `docker compose exec` works with the service name instead of the container name.